tools/build-jetpack.sh

```diff
@@ -56,6 +56,15 @@ if [[ $ADD_BETA_VERSION -eq 1 ]]; then
     echo "Now at version $CURRENT_VERSION!"
 fi
 
+# Checking for composer
+hash composer 2>/dev/null || {
```

Comment:
Looks like we could try to keep that DRY and move it to its own script that we could reuse.

Reply:
Yeah, the build scripts need to be DRYer. I don't think that has to be part of this PR since they're all triplicating everything (all the yarn commands, etc) right now.

Reply:
Sounds good to me 👍